docs/notifications.json

```diff
@@ -10,8 +10,13 @@
     "text": "You can <a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MUI_hq\">follow us on Twitter</a> to receive exclusive tips and updates about MUI and the React ecosystem."
   },
   {
-    "id": 58,
-    "title": "🚀 <b>The 2021 MUI developer survey results are out</b> — here's what we discovered",
-    "text": "Your feedback helps us to build better products. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"2021-developer-survey-results/\" href=\"/blog/2021-developer-survey-results/\">Go to the post to</a> find out what we learned about your needs in our annual survey."
+    "id": 62,
+    "title": "📣 <b>Date and time pickers have been moved to MUI X</b>",
+    "text": "<a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"lab-date-pickers-to-mui-x/\" href=\"/blog/lab-date-pickers-to-mui-x/\">Go to the post to</a> find out about the reasons behind this effort and how to migrate from lab to MUI X."
+  },
+  {
+    "id": 63,
+    "title": "📣 <b>Documentation per product is live!</b>",
+    "text": "<a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"docs-restructure-2022/\" href=\"/blog/docs-restructure-2022/\">Check out</a> all the changes and improvements."
```

Comment:
```suggestion
    "text": "Each of MUI's products now has its own dedicated documentation, making it easier than ever to find exactly what you need. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"docs-restructure-2022/\" href=\"/blog/docs-restructure-2022/\">Check out the blog</a> post for more details."
```

Reply:
Hey @danilo-leal I did change the URL of the page to `/blog/docs-restructure-2022/` to be explicit and would help on the SEO as well (in case we have other docs updates & improvements in the future). I hope this is fine for you.

Reply:
Any thoughts? @samuelsycamore

Reply:
Looks good to me! We probably don't need to worry too much about optimizing the slug for SEO on a post like this, so short and sweet is perfect. 👍